### ISSUE TITLE
[Bug] Deteched I/O inside the event loop

### ISSUE THREAD

**Issue by tcallaw** [JoshuaMulliken/ha-wyzeapi#125]
I installed this repository via HACS today. It is not working and I am getting a warning saying:
Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for wyzeapi doing I/O at custom_components/wyzeapi/smartbridge/providers/wyze/client.py, line 245: response = session.send(request, **settings)

Thanks in advance for your help

**To Reproduce**

**Expected behavior**
HA is not loading any of the bulbs

**System configuration**
System: <core-2021.3.4>
HA Version: <2021.3.4>
WyzeApi Version: <v0.5.15-vacuum-alpha.10> 

**configuration.yaml**

```yaml
wyzeapi:
   username: 
   password: 
   sensors: false
```

**home-assistant.log**
```
Logger: homeassistant.util.async_
Source: util/async_.py:129
First occurred: 4:42:32 PM (1 occurrences)
Last logged: 4:42:32 PM

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for wyzeapi doing I/O at custom_components/wyzeapi/smartbridge/providers/wyze/client.py, line 245: response = session.send(request, **settings)
```

**Comment by JoshuaMulliken** [comment]
Yes, this is an issue with the alpha version. Please use the latest stable. It is an issue with using some sync code inside of the async event loop.

**Comment by tcallaw** [comment]
Awesome...thanks. Can you confirm which is the stable version, is it
"Master"

This is what I see when I go to install

[image: Screen Shot 2021-03-31 at 12.15.44 PM.png]
